x-pack/plugins/cases/server/client/cases/get.ts

```diff
@@ -34,6 +34,12 @@ export const get = async ({
   includeSubCaseComments = false,
 }: GetParams): Promise<CaseResponse> => {
   try {
+    if (!ENABLE_CASE_CONNECTOR && includeSubCaseComments) {
```

Comment:
Moved this into the client so it throws if the client is used via the plugin contract in addition to the exposed route.